### PR TITLE
Add aws_limits entity

### LINKED ISSUE #78: Sync AWS resource limits
Some checks/alert could use these values to raise alerts/notifications once resources count is approaching set limits.

Not clear yet how each AWS resource/service expose those limits, but an example would be similar to https://github.com/jantman/awslimitchecker

### PR DESCRIPTION
**Pull request by mohabusama**
- EC2 used and max
- RDS
- ASG
- Some IAM limits

Note: we can add more limits based on requirements, so it is not considered to be a complete solution.

Resolve #78

### CODE REVIEW

File: zmon_aws_agent/aws.py [hunk 1]
```diff
@@ -679,3 +679,72 @@ def get_apps_from_entities(instances, account, region):
...
+
+        limits['iam-used-policies'] = iam_limits['Policies']
+        limits['iam-max-policies'] = iam_limits['PoliciesQuota']
+
+    except Exception as e:
+        logger.error('Failed to query IAM account summary {}'.format(e))
+
+    entity = {
+        'id': entity_id('aws-limits[{}:{}]'.format(acc, region)),
+        'type': 'aws_limit',
```
Comment (Jan-M) [inline zmon_aws_agent/aws.py]: Can this become "limits"? if id is plural

Comment (mohabusama) [inline zmon_aws_agent/aws.py]: Yep

**Comment by Jan-M** [comment]
:+1:

**Comment by mohabusama** [comment]
👍

**Pull request merged**